extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java

```diff
@@ -174,6 +175,7 @@ public TaskGroup(ImmutableMap<Integer, Long> partitionOffsets, Optional<DateTime
   private final KafkaTuningConfig taskTuningConfig;
   private final String supervisorId;
   private final TaskInfoProvider taskInfoProvider;
+  private final long futureTimeout; // how long to wait for async operations to complete
```

Comment:
include "in seconds"; millis is more common so people will assume that if you leave the units off.